src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
>  It would be akin to us crashing on some thread and then asking the user "running code asynchronously failed, do you want us to run it synchronously". It's not something they have the context to make a decision on, nor can they understand the implication of their choice. 

Unless they went to task manager and killed the process (which atleast i do a lot).  We don't have to show the UI to the user but crashing VS in this case seems bad to me. We could atleast catch and swallow this exception. HeeJae tells me that we can distinguish between the process going down because of an exception thrown in that process versus it disappearing without reason (most likely user killed it).